clients/geth/specular/rollup/utils/fmt/fmt.go

```diff
@@ -0,0 +1,28 @@
+package fmt
+
+import (
+	"fmt"
+	"runtime"
+	
+	"github.com/specularl2/specular/clients/geth/specular/rollup/utils/errors"
+)
+
+type wrapError struct {
+	err  *errors.Error
+	msg string
+}
+
+func Errorf(format string, args ...interface{}) *wrapError {
+	pc, _, line, _ := runtime.Caller(1)
+	fn := runtime.FuncForPC(pc)
+	err := errors.Errorf(format, args...)
+	return &wrapError{
```

Comment:
do we really need to wrap the error here—`errors.Errorf` already wraps it; can't we just concat with `format` before passing it in L18?

Reply:
actually on second thought, looking at the output the information it's appending here seems completely redundant; e.g.
```
/Users/ayush/hello/main.go:35 (0x100a8fadc)
        c: return fmt.Errorf("assertion failed for c: %w", err)
 | main.c:35                      <---- this is already mentioned on the first line
```

i suggest getting rid of everything here and renaming the `errors` package to `fmt` so we can just call `Errorf` directly from there

Reply:
ah got it can you just modify directly:
```
func (err *Error) Error() string {

	msg := err.Err.Error()
	if err.prefix != "" {
		msg = fmt.Sprintf("%s: %s", err.prefix, msg)
	}
	<specular modification>
	msg = msg + "\n" + string(err.Stack())
	<specular modification/>
	return msg 
}
```
and maybe remove ErrorStack then since it's redundant

and in the files borrowed we should attribute to go-errors (can you find what the correct licensing text is, their repo is MIT i believe)

Reply:
The changes have been implemented successfully.